### PR TITLE
feat(kvStore):Replace go-cache with sqlite

### PR DESCRIPTION
**Pull request by EMQmyd**

### CODE REVIEW

File: common/migratio.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package common
```
Comment (ngjaying) [inline common/migratio.go:1]: typo in file name. Should be migration

File: common/migratio_test.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package common
```
Comment (ngjaying) [inline common/migratio_test.go:1]: typo in file name

File: go.mod [hunk 3]
```diff
@@ -16,6 +16,7 @@ require (
 	github.com/keepeye/logrus-filename v0.0.0-20190711075016-ce01a4391dd1
 	github.com/lestrrat-go/file-rotatelogs v2.4.0+incompatible
 	github.com/lestrrat-go/strftime v1.0.3 // indirect
+	github.com/mattn/go-sqlite3 v1.14.5
 	github.com/patrickmn/go-cache v2.1.0+incompatible
```
Comment (ngjaying) [inline go.mod:20]: Can we remove go cache?

File: xstream/server/server/ruleManager.go [hunk 4]
```diff
@@ -69,6 +69,7 @@ func doStartRule(rs *RuleState) error {
 				tp.GetContext().SetError(err)
 				logger.Printf("closing rule %s for error: %v", rs.Name, err)
 				tp.Cancel()
+				rs.Triggered = false
```
Comment (ngjaying) [inline xstream/server/server/ruleManager.go:72]: Is this done in another PR?

**Review by ngjaying — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (EMQmyd) [reply go.mod:20]: Need to use gocache to migrate data

File: xstream/server/server/ruleManager.go (hunk 4 above)
Comment (EMQmyd) [reply xstream/server/server/ruleManager.go:72]: no The state of the rule needs to be changed when the rule is stopped

File: common/migratio_test.go (hunk 2 above)
Comment (EMQmyd) [reply common/migratio_test.go:1]: ok

File: common/migratio.go (hunk 1 above)
Comment (EMQmyd) [reply common/migratio.go:1]: ok

**Review by ngjaying — APPROVED** [review]
